Uplift/Helpers/OnboardingAnimations.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  OnboardingAnimations.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/26/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Presentation
+import UIKit
+
+// MARK: - Arrow Button Transition
+
+/// Transition that changes arrow colors when swiping
+public class ArrowButtonTransition: NSObject, Animatable {
+
+    private let content: Content
+    private let duration: TimeInterval
+    private let delay: TimeInterval
+    private var played = false
+    /// Used for the first arrow: turns yellow always when doing playBack
+    private var isFirstArrow = false
+
+    var transitionIsEnabled = false
+
+    public init(content: Content,
+                duration: TimeInterval = 1.0,
+                delay: TimeInterval = 0.0,
+                firstArrow: Bool = false) {
+        self.content = content
+        self.duration = duration
+        self.delay = delay
+
+        /// Animation for first arrow in sequence
+        /// When transitioning backwards, should always be enabled
+        if firstArrow {
+            isFirstArrow = firstArrow
+            transitionIsEnabled = true
+        }
+
+        super.init()
+    }
+
+    private func animate(_ towardsEnabled: Bool) {
+        UIView.animate(
+            withDuration: duration,
+            delay: delay,
+            options: [.beginFromCurrentState, .allowUserInteraction],
+            animations: ({ [unowned self] in
+                self.content.view.backgroundColor = towardsEnabled ? .primaryYellow : .none
+            }),
+            completion: { [unowned self] completedAnimation in
+                if let button = self.content.view as? UIButton, completedAnimation {
+                    button.isEnabled = self.transitionIsEnabled
+                }
+                self.played = false
+            }
+        )
+
+        played = true
+    }
+
+}
+
+extension ArrowButtonTransition {
+
+    public func play() {
+        if content.view.superview != nil {
+            if !played { animate(transitionIsEnabled) }
```

Comment:
you can combine these if statements into one:
```
if content.view.superview != nil && !played { ... }
```
same for the other ones in this file